src/object/get.js

```diff
@@ -24,21 +24,24 @@ module.exports = (send) => {
       options = {}
     }
 
+    let cid, b58Hash
```

Comment:
We default to declare each variable in separate lines. Mind replacing for:

```
let cid
let b58Hash
```

Also, with the added support to cid, makes sense to call it cidB58Str